early-init.el

```diff
@@ -0,0 +1,13 @@
+;; Before Emacs 27, the init file was responsible for initializing the package
+;; manager by calling `package-initialize'.  Emacs 27 changed the default
```

Comment:
There are two spaces before the word `Emacs` on this line.

Reply:
I'm a traditionalist!... Fixed.